### PR TITLE
Replace the datepicker with the `<AuDateInput>` component

### PR DESCRIPTION
**Pull request by Windvis**
This makes it easier for users to type in the date which will speed up data entry, especially if dates are far away in the past or future.

### CODE REVIEW

**Review by Poltergeistz — APPROVED** [review]
Everything works as expected :+1:

File: app/components/leidinggevendenbeheer/functionaris-form.hbs [hunk 1]
```diff
@@ -8,21 +8,21 @@
     <div class="au-o-grid au-o-grid--tiny">
       <div class="au-o-grid__item au-u-1-2 au-u-1-3@medium">
         <AuLabel for="date-picker-start">Start aanstellingsperiode</AuLabel>
-        <AuDatePicker
+        <AuDateInput
           @value={{@model.start}}
           @onChange={{fn this.handleDateChange "start"}}
```
Comment (Windvis) [inline app/components/leidinggevendenbeheer/functionaris-form.hbs:13]: I need to double check if this also doesn't need the `@width="block"` arg.

Comment (Poltergeistz) [reply app/components/leidinggevendenbeheer/functionaris-form.hbs:13]: Sure, I'll check

Comment (Windvis) [reply app/components/leidinggevendenbeheer/functionaris-form.hbs:13]: It seems the previous version was filling the width, so I'll add that here as well:

image

Comment (Poltergeistz) [reply app/components/leidinggevendenbeheer/functionaris-form.hbs:13]: It works well without it. There is something I've noticed for small devices, if you don't provide the `block` arg the two input will overlap a bit. And for larger resolution they will be far from each other. 

So I think it's better for UX to add this argument.

**Comment by Windvis** [comment]
Thanks for the review 👍

**Pull request merged**